mesheryctl/internal/cli/root/system/check.go

```diff
@@ -477,65 +488,47 @@ func (hc *HealthChecker) runComponentsHealthChecks() error {
 	if hc.Options.PrintLogs {
 		log.Info("\nMeshery Components \n--------------")
 	}
-	return hc.runAdapterHealthChecks()
+	return hc.runAdapterHealthChecks("")
+}
+
+// runOperatorHealthChecks executes health-checks for Operators
+func (hc *HealthChecker) runOperatorHealthChecks() error {
+	//TODO
+	return nil
 }
 
 // runAdapterHealthChecks executes health-checks for Adapters
-func (hc *HealthChecker) runAdapterHealthChecks() error {
+// If no adapter is specified all the adapters are checked
+func (hc *HealthChecker) runAdapterHealthChecks(adapter_name string) error {
```

Comment:
pll fix the lint-error

Reply:
@deepakdinesh1123 you might have outstanding feedback here.

Reply:
I was thinking of demoing this first to get everyone's thoughts and then get back to fixing it. I also have another item to discuss along with this.